examples/fizzbuzz/Forc.lock

```diff
@@ -4,8 +4,9 @@ dependencies = []
 
 [[package]]
 name = 'fizzbuzz'
-dependencies = ['std']
+dependencies = ['std git+https://github.com/fuellabs/sway?tag=v0.9.2#4905aa0b2b6cb7178d6229e7bed94c241a418c26']
```

Comment:
Should keep the local path `std` for these example projects, otherwise they're not actually being tested as part of CI.

Reply:
When I removed the dependency in `Forc.toml`, this happened.. I suppose `std` was automatically being added which uses the Github link. I'll revert the change in `Forc.toml`.